src/components/ConfessIt.jsx

```diff
@@ -99,7 +109,11 @@ const ConfessIt = () => {
         </SwiperSlide>
         <SwiperSlide>
           <Column title={t("sins_list.review", "Review")}>
-            <SinsList sinsList={sinsList} onRemoveSinItem={removeSinItem} />
+            <SinsList
+              key={sinsListVersion}
```

Comment:
This is just so React knows the component changed when we remove sins? Should we be using a `DndContext` instead?